slash-commands/slash-commands.js

```diff
@@ -115,14 +149,23 @@ CandyShop.SlashCommands = (function(self, Candy, $) {
 	self.part = function() {
 		Candy.Core.Action.Jabber.Room.Leave(self.currentRoom());
 	};
+	
+	/** Function: leave
+	 * Exits the current chat room
+	 *
+	 */
+	self.leave = function() {
```

Comment:
Really don't like the provided syntax, But I will DRY up \leave to just call \part.